templates/io.prometheus.node_exporter.plist.j2

```diff
@@ -31,6 +31,6 @@
   <key>StandardOutPath</key>
   <string>/tmp/{{ node_exporter_ident }}.stdout</string>
   <key>UserName</key>
-  <string>{{ ansible_user }}</string>
+  <string>{{ node_exporter_user }}</string>
```

Comment:
Please reword https://github.com/Ableton/ansible-role-prometheus-node-exporter/blob/ee706620aff6c8747c11e8733618bfe657276152/defaults/main.yml#L24-L25

Reply:
Fixed in 7cf482b.